### PR TITLE
send coverage results to codacy after every successful ci build

### PR DESCRIPTION
**Pull request by erciccione**
Result:

coverage

The detailed coverage stats will be on https://app.codacy.com/gh/haveno-dex/haveno/dashboard. I will add a badge on the readme after this PR is merged (it's not available until the first report is sent)

### CODE REVIEW

**Comment by erciccione** [comment]
Build fails but i think that's expected, since the configuration is not merged yet.

**Review by premek — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,6 +28,11 @@ jobs:
         run: git lfs pull
       - name: Build with make
         run: make
+      - name: Send coverage report to Codacy
+        env:
+          CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        if: success()
+        run: bash <(curl -Ls https://coverage.codacy.com/get.sh) report -r assets/build/reports/jacoco/test/jacocoTestReport.xml
```
Comment (premek) [inline .github/workflows/build.yml:35]: maybe you could add this also to the build-windows branch below

**Comment by premek** [comment]
>Build fails but i think that's expected, since the configuration is not merged yet.

Is CODACY_PROJECT_TOKEN configured in repo settings under Secrets? Or where is the value taken from?

File: .github/workflows/build.yml (hunk 1 above)
Comment (erciccione) [reply .github/workflows/build.yml:35]: Sure, done! I wasn't sure if the bash command would have worked.

**Comment by erciccione** [comment]
> Is CODACY_PROJECT_TOKEN configured in repo settings under Secrets? Or where is the value taken from?

It's in the repo settings under Secrets, yes. I tried this setup on my own fork first and had no issues (the screenhsot comes from the coverage analysis on my fork), should work here too after it's merged (I changed the value of `CODACY_PROJECT_TOKEN` accordingly)

**Comment by premek** [comment]
you're right, it's not working, I didn't realized that, sorry,
I just saw the windows build passing and wondered why

**Comment by premek** [comment]
probably the secrets are not available in pull requests which makes perfect
sense when I think about it now :)

File: .github/workflows/build.yml (hunk 1 above)
Comment (premek) [reply .github/workflows/build.yml:35]: you're right, it's not working, I didn't realized that, sorry,
I just saw the windows build passing and wondered why

**Pull request merged**